.github/workflows/bump-version-and-create-release.yaml

```
@@ -96,3 +97,21 @@ jobs:
           tag: ${{ needs.compute-version.outputs.new_tag }}
           body: ${{ github.event.pull_request.body }}
 
+
+  build-and-run-release-tests:
+    needs: [compute-version, create-release]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build release test Docker image
+        uses: docker/build-push-action@v6
+        with:
+          build-args: |
+            RELEASE_DOWNLOAD_URL=https://github.com/KIT-MRT/util_caching/releases/download/${{ needs.compute-version.outputs.new_tag }}
+          push: false
+          tags: |
+            ghcr.io/kit-mrt/util_caching_release_tests
```

See also the corresponding suggestion below.

I named the docker image to something more generic above to be able to copy paste this workflow to the arbitration graphs repo. The registry name is also not required since we don't plan to push it.
But up to you, I'm fine either way. 
With the default build argument for the download url, it isn't 100% generic anymore anyway (and it doesn't really need to be) :shrug: 

```suggestion
            release_tester
```

---

Good idea!
I defined the default/latest download url in the dockerfile to allow for local testing :wink:

---

I tried to do a more generic setup by using the `html_url` output of the ncipollo/release-action (it points to the new release). But then it's a mess to support both download URL formats:

https://github.com/KIT-MRT/util_caching/releases/download/v1.2.0/libmylib-dev.deb
https://github.com/KIT-MRT/util_caching/releases/latest/download/libmylib-dev.deb

Beware the swapped position of `download/` :facepalm:

---

See 417ec26